### DISCUSSION TITLE
Help Setting Up Wyze Cam v3 (Hangs on Red Light)

### DISCUSSION THREAD

**Discussion by kingkangaroo** [gtxaspec/wz_mini_hacks#501] [Q&A] [answered]
I just got a Wyze Cam v3 with firmware version 4.36.0.150.
I have been trying to install wz mini hacks since yesterday, but have not had any success.  I am able to hear the voice say that it is unpacking the files, but my camera's light stays red and rebooting doesn't fix anything.

I had not had success using files where I changed the configuration, so I am currently trying to install by using the default SD Root files, unmodified.  I am using:

- Windows 11 Laptop
- Microcenter Brand 32GB SD Card (Formatted FAT32 via Windows formatter and SDCard.org formatter, have tried both Quick Format and Normal Format)

From the main GitHub page, I click on the green Code button and select Download Zip as shown here:
https://i.imgur.com/E1sBj62.jpg

I unzip the archive using WinRAR and copy only the contents of the SD_Root folder to the root of my newly-formatted FAT32 SD Card as shown here:
https://i.imgur.com/z8B2FBq.jpg

And this is where I am getting stuck.  I am not modifying any of the files, it just hangs with a red light when trying to install.
Does anybody know where I might be going wrong?

**Comment by Bob-AOL** [comment, marked as answer]
Start over by upgrading the firmware to 4.36.9.139 which is the last known compatible version. It's also probably what it would upgrade to by using the Wyze app as I think the latest firmware mess they rolled out was rolled back and is still a mess.

Reformat the SD card just using Windows..., or try a new SD card. A Microcenter card will be correctly formatted right out of the package.

Unzip the distro again just using the Windows 'extract' function. Copy the required files to the SD card and give it another spin.

My money is on the SD card. But if you're lucky, just upgrading the firmware will do the trick. On the other hand, it could also be the card reader. You'd find that out by manually upgrading the firmware instead of doing it online via the Wyze app. May want to consider that.

Nothing fancy required. Mini-Hacks is about as Windows friendly as it gets.

**Reply by kingkangaroo** [reply]
Thanks Bob!  Upgrading the firmware seems to have done the trick!
Everything is working great now 👍

**Reply by Bob-AOL** [reply]
Glad to hear it and happy to help.
